packages/api/tests/endpoints/test-executions.js

```diff
@@ -684,3 +709,89 @@ test.serial('POST /executions/search-by-granules returns 400 when the Metrics EL
 
   t.regex(response.body.message, /ELK Metrics stack not configured/);
 });
+
+test.serial('POST /executions/workflows-by-granules returns correct executions when granules array is passed', async (t) => {
+  const { collectionId, fakeGranules, fakePGExecutions } = t.context;
+
+  const response = await request(app)
+    .post('/executions/workflows-by-granules?limit=10')
```

Comment:
The `limit` doesn't work.  When I set it to 1, the endpoint still returns all the workflows (the granule only has 2 executions though). I actually want all the workflowNames returned, or the limit to be higher number.